packages/@lwc/template-compiler/src/__tests__/fixtures/attributes/valid-hyphen-underscore/actual.html

```diff
@@ -3,4 +3,7 @@
 <template>
     <!-- attribute name with hyphen + underscore "-_" combination -->
     <x-button under-_hyphen="bar"></x-button>
+
+    <!-- attribute name with leading hyphen + underscore "-_" combination -->
+    <x-button -_leading="bar"></x-button>
```

Comment:
Can you also add the following case? This case is from the [usage data](https://docs.google.com/spreadsheets/d/14yJbgk7VxIiD56PiGvegjGF4a6JcVqTOM5fuNlTbCCQ/edit#gid=1028394080).

`<x-button foo_-bar="bar"></x-button>`